common/src/main/java/com/codeforcommunity/enums/ApprenticeshipProgram.java

```diff
@@ -1,9 +1,10 @@
 package com.codeforcommunity.enums;
 
 public enum ApprenticeshipProgram {
```

Comment:
For the sake of consistency, is there any reason why these aren't `OECS_YES`, `OTHER`, `NO_APPRENTICESHIP_PROGRAM`?